_docs_integrate/error-codes.md

```diff
@@ -10,16 +10,88 @@ toc: true
 properties:
   - id: SC089
   - category: Troubleshooting
-  - description: https//enmeshed.eu/integrate/error-codes
+  - description: https://enmeshed.eu/integrate/error-codes
```

Comment:
I would rather not insert any links under 'description', this only happens in about 4 places in the documentation. I would either leave this blank or insert text. (Here and in the Excel)